### PR TITLE
Referencing fields

### PR DESCRIPTION
**Pull request by hariso**
Part of https://github.com/ConduitIO/conduit/issues/1422, includes specification changes from https://github.com/ConduitIO/conduit/pull/1424.

### CODE REVIEW

**Review by raulb — APPROVED** [review]
Nice! @hariso Great call already pointing out to the page that will be added as part of https://github.com/ConduitIO/conduit-site/pull/49. Will make sure I ship that one soon so we can ship this one without a broken link.

File: docs/processors/referencing-fields.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: 'Referencing record fields'
+sidebar_position: 1
+---
+
+Very often, processors can be configured to work on a specific field in a
+record (which is always in
+the [OpenCDC format](/docs/features/opencdc-record.mdx)). That is done through
```
Comment (raulb) [inline docs/processors/referencing-fields.mdx:8]: I just realized about this.

```suggestion
the [OpenCDC format](/docs/features/opencdc-record)). That is done through
```

**Review by raulb — CHANGES_REQUESTED** [review]
I just realized that we need to link differently to this page.

File: docs/processors/referencing-fields.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: 'Referencing record fields'
+sidebar_position: 1
+---
+
+Very often, processors can be configured to work on a specific field in a
```
Comment (maha-hajja) [inline docs/processors/referencing-fields.mdx:6]: I think we should also mention that these references are how the fields would be addressed in a Go template, and maybe add a link for go templates format

**Review by raulb — APPROVED** [review]
Looks great. I really like having both the json and the dot notation side by side, extending this information below. Great work 👌

**Pull request merged**